pkg/volume/volume.go

```diff
@@ -168,7 +168,7 @@ type Detacher interface {
 	// UnmountDevice unmounts the global mount of the disk. This
 	// should only be called once all bind mounts have been
 	// unmounted.
-	UnmountDevice(deviceMountPath string) error
+	UnmountDevice(deviceMountPath string) (string, error)
```

Comment:
you need to at least document what this second return is. Not at all obvious
 Add a line note